src/components/InputTime/AsString.tsx

```diff
@@ -65,6 +66,34 @@ const AsString: React.FC<AsStringProps> = ({
   // value is the true, controlled value, patched for undefined / empty
   const value = useMemo(() => valueOrUndefined || '', [valueOrUndefined]);
 
+  const step: number | undefined = useMemo(() => {
+    let safeStep: number | undefined;
+    safeStep = typeof customStep === 'number' ? customStep : safeStep;
```

Comment:
this looks a little bit confusing, what about using a switch statement instead?

Reply:
This section was pretty pointless. I'm going to just `parseInt(customStep, 10)` for all cases.

`parseInt(customStep, 10)`  works on numbers just fine.
`parseInt(undefined, 10)` produces `NaN`, which is explicitly guarded against in the next block.

Reply:
Would you have another look when you get a chance, @sebastianvera?